python/cac_tripplanner/templates/base.html

```diff
@@ -69,6 +69,11 @@
 
     {% endblock %}
 
+    <script type="text/javascript">
```

Comment:
That's a good point, I'll move it. We may want to move it back here if we end up using the settings on other pages.